sdk/servicebus/Azure.Messaging.ServiceBus/tests/Amqp/AmqpConverterTests.cs

```diff
@@ -156,5 +156,24 @@ public void CanParseMaxAbsoluteExpiryTime()
 
             Assert.AreEqual(DateTimeOffset.MaxValue, convertedSbMessage.ExpiresAt);
         }
+
+        [Test]
+        public void CanParseGuidBytes()
+        {
+            var input = Guid.NewGuid();
+
+            Guid output = AmqpMessageConverter.ParseGuidBytes(input.ToByteArray());
+            Assert.AreEqual(input, output);
+        }
+
+        [Test]
+        public void CanWriteGuidBytes()
```

Comment:
@jsquire I might have to take you up on your offer to run these on a big endian machine 😄